Makefile

```diff
@@ -78,6 +78,21 @@ gdbstub-test: $(BIN)
 	$(Q).ci/gdbstub-test.sh && $(call notice, [OK])
 endif
 
+ENABLE_JIT ?= 1
+$(call set-feature, JIT)
+ifeq ($(call has, JIT), 1)
+CFLAGS += -I./mir
+LDFLAGS += mir/libmir.a -lpthread
+
+OBJS_EXT += codegen.o compile.o
+
+mir/GNUmakefile:
+	git submodule update --init $(dir $@)
+	
+mir/libmir.a: mir/GNUmakefile
```

Comment:
For the consistency of source code layout, place MIR directory under `src`. That is, specify `src/mir` here.